branca/utilities.py

```diff
@@ -164,61 +170,97 @@ def color_brewer(color_code, n=6):
+               'PiYG': ['#c51b7d', '#e9a3c9', '#fde0ef','#f7f7f7',
                         '#e6f5d0', '#a1d76a', '#4d9221'],
-               'PRGn': ['#762a83', '#af8dc3', '#e7d4e8',
+               'PRGn': ['#762a83', '#af8dc3', '#e7d4e8','#f7f7f7',
                         '#d9f0d3', '#7fbf7b', '#1b7837'],
-               'PuOr': ['#b35806', '#f1a340', '#fee0b6',
+               'PuOr': ['#b35806', '#f1a340', '#fee0b6','#f7f7f7',
                         '#d8daeb', '#998ec3', '#542788'],
-               'RdBu': ['#b2182b', '#ef8a62', '#fddbc7',
+               'RdBu': ['#b2182b', '#ef8a62', '#fddbc7','#f7f7f7',
                         '#d1e5f0', '#67a9cf', '#2166ac'],
-               'RdGy': ['#b2182b', '#ef8a62', '#fddbc7',
+               'RdGy': ['#b2182b', '#ef8a62', '#fddbc7','#ffffff',
                         '#e0e0e0', '#999999', '#4d4d4d'],
-               'RdYlBu': ['#d73027', '#fc8d59', '#fee090',
+               'RdYlBu': ['#d73027', '#fc8d59', '#fee090','#ffffbf',
                           '#e0f3f8', '#91bfdb', '#4575b4'],
-               'RdYlGn': ['#d73027', '#fc8d59', '#fee08b',
+               'RdYlGn': ['#d73027', '#fc8d59', '#fee08b','#ffffbf',
                           '#d9ef8b', '#91cf60', '#1a9850'],
-               'Spectral': ['#d53e4f', '#fc8d59', '#fee08b',
+               'Spectral': ['#d53e4f', '#fc8d59', '#fee08b','#ffffbf',
                             '#e6f598', '#99d594', '#3288bd'],
                'Accent': ['#7fc97f', '#beaed4', '#fdc086',
-                          '#ffff99', '#386cb0', '#f0027f'],
+                          '#ffff99', '#386cb0', '#f0027f','#bf5b17'],
                'Dark2': ['#1b9e77', '#d95f02', '#7570b3',
-                         '#e7298a', '#66a61e', '#e6ab02'],
+                         '#e7298a', '#66a61e', '#e6ab02','#a6761d'],
                'Paired': ['#a6cee3', '#1f78b4', '#b2df8a',
-                          '#33a02c', '#fb9a99', '#e31a1c'],
+                          '#33a02c', '#fb9a99', '#e31a1c','#fdbf6f'],
                'Pastel1': ['#fbb4ae', '#b3cde3', '#ccebc5',
-                           '#decbe4', '#fed9a6', '#ffffcc'],
+                           '#decbe4', '#fed9a6', '#ffffcc','#e5d8bd'],
                'Pastel2': ['#b3e2cd', '#fdcdac', '#cbd5e8',
-                           '#f4cae4', '#e6f5c9', '#fff2ae'],
+                           '#f4cae4', '#e6f5c9', '#fff2ae','#f1e2cc'],
                'Set1': ['#e41a1c', '#377eb8', '#4daf4a',
-                        '#984ea3', '#ff7f00', '#ffff33'],
+                        '#984ea3', '#ff7f00', '#ffff33','#a65628'],
                'Set2': ['#66c2a5', '#fc8d62', '#8da0cb',
-                        '#e78ac3', '#a6d854', '#ffd92f'],
+                        '#e78ac3', '#a6d854', '#ffd92f','#e5c494'],
                'Set3': ['#8dd3c7', '#ffffb3', '#bebada',
-                        '#fb8072', '#80b1d3', '#fdb462'],
+                        '#fb8072', '#80b1d3', '#fdb462','#b3de69'],
```

Comment:
I wonder if we should factor out the colormaps to another file to make them easier to manage...

Reply:
I think that makes sense as well. Also the way colorbrewer works you have different end-points for color that jump between 5 to 7 and then again at 11 so if we want to follow it 100% we will need a few different list definitions depending on desired length, or we can just pick one.